src/content/reference/react/useRef.md

```diff
@@ -92,19 +92,19 @@ function handleStopClick() {
 }
 ```
 
-By using a ref, you ensure that:
+Dengan menggunakan ref, Anda memastikan bahwa:
 
-- You can **store information** between re-renders (unlike regular variables, which reset on every render).
-- Changing it **does not trigger a re-render** (unlike state variables, which trigger a re-render).
+- Anda dapat **menyimpan informasi** di antara render ulang (tidak seperti variabel biasa, yang disetel ulang pada setiap render).
+- Mengubahnya **tidak memicu sebuah render ulang** (tidak seperti variabel state, yang memicu sebuah render ulang).
 - The **information is local** to each copy of your component (unlike the variables outside, which are shared).
```

Comment:
Kurang bagian ini, kelewatan belum diterjemahkan:

```
- The **information is local** to each copy of your component (unlike the variables outside, which are shared).
```

Reply:
```suggestion
- **Informasinya lokal** untuk setiap salinan komponen Anda (tidak seperti variabel di luar, yang dibagikan).
```

Reply:
@xevorte It's okay too, `Informasi ini bersifat lokal` should be fine.

Prior I suggested that because the source doesn't have `this` word. It wasn't `This **information is local**` but rather `The **information is local**`.